### PR TITLE
Emerson-fernandes/challenge-html

### PR DESCRIPTION
**Pull request by Emerson-feral**

### CODE REVIEW

**Review by IsabelGomez87 — APPROVED** [review]
ok!

File: emerson-fernandes/precourse/challenge/semanthic-html/news.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                <h4>New Documentary Creates a Buzz</h4>
+                <p>Embracing and moving towards discovering valuable truths midwifery, carob shamanic ceremony. Forest honoring your truth elder ancient spiritual traditions, new paradigm projecting retreat. Holographic resonance double rainbow multi-dimensional,
+                    nature core strength agave juicy braggs radiant farm to table reiki master. Pranayama entheogen fractal nature of, candida cleanse ecofriendly sarong lingham hacky sack.</p>
+                <p>Aho light energy healing energy, lingham manifestation raw cacao seasonal float tank. Sacred geometric shapes tincture embracing and moving towards quinoa tofu peace, sun salutation elder shakti. Dance dome incense holographic resonance
+                    safe space cuddle party soul-level contract radiant harmful electromagnetic fields, loving kindness tantra. Sound healing spirit animal meditation bodyworker, breatharian vatta imbalance. Consciousness co-op spiritual awakening, as
+                    we honor third eye beltane. Dreamwork white sage infinite blessings empowering messages enneagram, bioneers spinal awareness. Namaste.
+
```
Comment (dmartorell) [inline emerson-fernandes/precourse/challenge/semanthic-html/news.html:57]: Sobran espacios aquí

**Review by dmartorell — CHANGES_REQUESTED** [review]

**Review by jasuarez21 — DISMISSED** [review]
ni neymar, ronaldinho y ronaldo, programan asi!

**Review by annasm07 — DISMISSED** [review]

**Review by CheCabeza — DISMISSED** [review]

**Review by gteixi — APPROVED** [review]

File: emerson-fernandes/precourse/challenge/semanthic-html/styles/main.css [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
```
Comment (ulldecorb) [inline emerson-fernandes/precourse/challenge/semanthic-html/styles/main.css:21]: In css files, add line break after close each property for a good performance and better readability.

File: emerson-fernandes/precourse/challenge/semanthic-html/styles/main.css [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+/*        HTML5 display-role reset for older browsers         */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
```
Comment (ulldecorb) [inline emerson-fernandes/precourse/challenge/semanthic-html/styles/main.css:26]: Caution: Add line break when close a property across all the file.

File: emerson-fernandes/precourse/challenge/semanthic-html/styles/responsive-nav.css [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+@media screen and (min-width: 980px) {
+  
+  .nav-toggle {
+    margin: 25px 0 0 10px;
+    display: block;
+  }
+  
+}
+
+  .nav-toggle:hover {
```
Comment (ulldecorb) [inline emerson-fernandes/precourse/challenge/semanthic-html/styles/responsive-nav.css:70]: Respect tab's hierarchy and delete not necessary space at begin of .nav-toggle:hover, a.nav-toggle & a.nav-toggle:hover

**Review by ulldecorb — COMMENTED** [review]
God job with sass, but a little mess with space.

**Review by gaxxrodri — APPROVED** [review]

**Review by gteixi — APPROVED** [review]

**Review by cristian-bars — APPROVED** [review]
Good job Emerson!

**Review by IrenePerezAlonso — APPROVED** [review]

**Review by gaxxrodri — APPROVED** [review]
💯

**Review by IsabelGomez87 — APPROVED** [review]

**Review by ulldecorb — APPROVED** [review]
Bom trabalho Emerson!!

**Pull request closed without merging**